src/pages/dashboard/AllLineschart/AllLinesChart.tsx

```diff
@@ -37,7 +38,7 @@ export const AllLinesChart: FC<AllChartComponentProps> = ({ startDate, endDate }
           title={convertLineFeedToHtmlTags(t('dashboard_tooltip_content'))}
           placement="left"
           arrow>
-          <span className="tooltip-icon">i</span>
```

Comment:
did you remove the relevant rule from the scss file?

Reply:
No. My bad. Will remove it asap